src/core/model/Stroke.h

```diff
@@ -116,6 +162,12 @@ class Stroke: public AudioElement {
 protected:
     void calcSize() const override;
 
+private:
+    static std::optional<Interval<double>> intersectLineWithRectangle(const Point& p, const Point& q,
```

Comment:
What is the meaning of Interval::min / max, how do they relate to p,q, rectangle? The optional is obvious.

Reply:
Will add a comment

Reply:
done